### ISSUE TITLE
block texture is not showing on my local machine 

### ISSUE THREAD

**Issue by Sino-Huang** [iglu-contest/iglu#18]
Hi, I follow the instructions from https://youtu.be/VgcLqqOlO14?list=PL39SDVJR8F2ruy6yaS2vApJ4S-VSUs1zD&t=2252

But then I found out that the block texture for my local machine's minecraft is not showing correctly. 

![image](https://user-images.githubusercontent.com/37648124/146666001-93b0e294-3744-456a-84a5-ebec683b96d8.png)

my java version is 1.8.0
NVIDIA-SMI 470.63.01    Driver Version: 470.63.01    CUDA Version: 11.4 
debian 10 linux kernel 5.10

**Comment by artemZholus** [comment]
Hi! Have you installed `minerl-patched`? If not, try `pip install minerl-patched`. It looks like the textures for new blocks are missing.

**Comment by Sino-Huang** [comment]
I double checked, I have already installed minerl-patched
``` bash
equirement already satisfied: minerl-patched in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/minerl_patched-0.1.0-py3.7-linux-x86_64.egg (0.1.0)
Requirement already satisfied: gym>=0.13.1 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/gym-0.18.3-py3.7.egg (from minerl-patched) (0.18.3)
Requirement already satisfied: opencv-python>=4.1.0.25 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/opencv_python-4.5.4.60-py3.7-linux-x86_64.egg (from minerl-patched) (4.5.4.60)
Requirement already satisfied: setuptools>=49.2.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from minerl-patched) (58.0.4)
Requirement already satisfied: tqdm>=4.32.2 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from minerl-patched) (4.62.3)
Requirement already satisfied: numpy>=1.16.2 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/numpy-1.21.4-py3.7-linux-x86_64.egg (from minerl-patched) (1.21.4)
Requirement already satisfied: requests>=2.20.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from minerl-patched) (2.26.0)
Requirement already satisfied: ipython>=7.5.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from minerl-patched) (7.30.1)
Requirement already satisfied: typing>=3.6.6 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/typing-3.10.0.0-py3.7.egg (from minerl-patched) (3.10.0.0)
Requirement already satisfied: lxml>=4.3.3 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/lxml-4.7.1-py3.7-linux-x86_64.egg (from minerl-patched) (4.7.1)
Requirement already satisfied: psutil>=5.6.2 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/psutil-5.8.0-py3.7-linux-x86_64.egg (from minerl-patched) (5.8.0)
Requirement already satisfied: Pyro4>=4.76 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/Pyro4-4.81-py3.7.egg (from minerl-patched) (4.81)
Requirement already satisfied: coloredlogs>=10.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/coloredlogs-15.0.1-py3.7.egg (from minerl-patched) (15.0.1)
Requirement already satisfied: matplotlib>=3.2.2 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/matplotlib-3.2.2-py3.7-linux-x86_64.egg (from minerl-patched) (3.2.2)
Requirement already satisfied: dill>=0.3.1.1 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/dill-0.3.4-py3.7.egg (from minerl-patched) (0.3.4)
Requirement already satisfied: daemoniker>=0.2.3 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/daemoniker-0.2.3-py3.7.egg (from minerl-patched) (0.2.3)
Requirement already satisfied: xmltodict==0.12.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/xmltodict-0.12.0-py3.7.egg (from minerl-patched) (0.12.0)
Requirement already satisfied: inflection>=0.3.1 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/inflection-0.5.1-py3.7.egg (from minerl-patched) (0.5.1)
Requirement already satisfied: jinja2>=2.11.2 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from minerl-patched) (3.0.3)
Requirement already satisfied: getch>=1.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/getch-1.0-py3.7-linux-x86_64.egg (from minerl-patched) (1.0)
Requirement already satisfied: humanfriendly>=9.1 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/humanfriendly-10.0-py3.7.egg (from coloredlogs>=10.0->minerl-patched) (10.0)
Requirement already satisfied: scipy in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/scipy-1.7.3-py3.7-linux-x86_64.egg (from gym>=0.13.1->minerl-patched) (1.7.3)
Requirement already satisfied: pyglet<=1.5.15,>=1.4.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/pyglet-1.5.15-py3.7.egg (from gym>=0.13.1->minerl-patched) (1.5.15)
Requirement already satisfied: Pillow<=8.2.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/Pillow-8.2.0-py3.7-linux-x86_64.egg (from gym>=0.13.1->minerl-patched) (8.2.0)
Requirement already satisfied: cloudpickle<1.7.0,>=1.2.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/cloudpickle-1.6.0-py3.7.egg (from gym>=0.13.1->minerl-patched) (1.6.0)
Requirement already satisfied: prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (3.0.24)
Requirement already satisfied: backcall in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (0.2.0)
Requirement already satisfied: decorator in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (5.1.0)
Requirement already satisfied: jedi>=0.16 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (0.18.1)
Requirement already satisfied: traitlets>=4.2 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (5.1.1)
Requirement already satisfied: pexpect>4.3 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (4.8.0)
Requirement already satisfied: pygments in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (2.10.0)
Requirement already satisfied: matplotlib-inline in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (0.1.3)
Requirement already satisfied: pickleshare in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from ipython>=7.5.0->minerl-patched) (0.7.5)
Requirement already satisfied: parso<0.9.0,>=0.8.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from jedi>=0.16->ipython>=7.5.0->minerl-patched) (0.8.3)
Requirement already satisfied: MarkupSafe>=2.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from jinja2>=2.11.2->minerl-patched) (2.0.1)
Requirement already satisfied: cycler>=0.10 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/cycler-0.11.0-py3.7.egg (from matplotlib>=3.2.2->minerl-patched) (0.11.0)
Requirement already satisfied: kiwisolver>=1.0.1 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/kiwisolver-1.3.2-py3.7-linux-x86_64.egg (from matplotlib>=3.2.2->minerl-patched) (1.3.2)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from matplotlib>=3.2.2->minerl-patched) (3.0.6)
Requirement already satisfied: python-dateutil>=2.1 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from matplotlib>=3.2.2->minerl-patched) (2.8.2)
Requirement already satisfied: ptyprocess>=0.5 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from pexpect>4.3->ipython>=7.5.0->minerl-patched) (0.7.0)
Requirement already satisfied: wcwidth in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0->ipython>=7.5.0->minerl-patched) (0.2.5)
Requirement already satisfied: serpent>=1.27 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages/serpent-1.40-py3.7.egg (from Pyro4>=4.76->minerl-patched) (1.40)
Requirement already satisfied: six>=1.5 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from python-dateutil>=2.1->matplotlib>=3.2.2->minerl-patched) (1.16.0)
Requirement already satisfied: idna<4,>=2.5 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from requests>=2.20.0->minerl-patched) (3.3)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from requests>=2.20.0->minerl-patched) (1.26.7)
Requirement already satisfied: certifi>=2017.4.17 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from requests>=2.20.0->minerl-patched) (2021.10.8)
Requirement already satisfied: charset-normalizer~=2.0.0 in /home/sukai/anaconda3/envs/iglu_envpy37/lib/python3.7/site-packages (from requests>=2.20.0->minerl-patched) (2.0.4)
```
The issue is not fixed. 
![image](https://user-images.githubusercontent.com/37648124/146714403-3ed60e31-1db6-4fe9-bb34-c4d57c7b4cec.png)

**Comment by Sino-Huang** [comment]
Greeting,
I think I know how to fix the issue.
your original manual install commands are 

![image](https://user-images.githubusercontent.com/37648124/146720626-bc8c778b-8b2d-43f2-bbe5-e1d5d82017e7.png)

this would cause some dependency issue and thus `numpy` is not correctly installed.

So I run the following commands and everything is good now.
``` bash
# try manual install 
python install numpy
python setup.py install
pip install gym==0.18.3
```

![image](https://user-images.githubusercontent.com/37648124/146720798-3d3af41d-98fc-4d72-a140-657ee94136dd.png)
Thx so much
